src/Cake/Commands/DefaultCommandSettings.cs

```diff
@@ -59,5 +59,9 @@ public sealed class DefaultCommandSettings : CommandSettings
         [CommandOption("--info")]
         [Description("Displays additional information about Cake.")]
         public bool ShowInfo { get; set; }
+
+        [CommandOption("--regeneratecache")]
```

Comment:
maybe `--invalidate-cache`?

Reply:
@bjorkstromm maybe `--force-cache-recompile` and `"Forces script to recompiled and cached (if caching is enabled)."`

?